services/motion/builtin/move_request.go

```diff
@@ -518,8 +518,44 @@ func (ms *builtIn) newMoveOnGlobeRequest(
 		return nil, err
 	}
 
+	// convert obstacles of type []GeoGeometry into []Geometry
 	geomsRaw := spatialmath.GeoGeometriesToGeometries(obstacles, origin)
 
+	// convert bounding regions which are GeoGeometries into Geometries
+	boundingRegions := req.BoundingRegions
+	if boundingRegions == nil {
+		boundingRegions = []*spatialmath.GeoGeometry{}
+	}
+	interactionSpaces := spatialmath.GeoGeometriesToGeometries(boundingRegions, origin)
```

Comment:
This is a little weird to me; how does the user decide the correct height for these obstacles?

For bounding regions, it sort of seems to me like the thing to do is to project all geometries to a 2d plane, and do collision detection there.

If I were a user defining a bounding region, I would expect that the correct Z dimension for my rectangle would be 0 and that it would do the right thing.